docs/build/micro-rdk/board/esp32.md

```diff
@@ -136,7 +136,7 @@ The following properties are available for `digital_interrupts`:
 
 ### PWM signals on `esp32` pins
 
-You can set PWM frequencies with Viam through the [`GPIOPin` API](/build/micro-rdk/board/#api).
+You can set PWM frequencies with Viam through the [`GPIOPin` API](/build/micro-rdk/board/#api). Make sure to check [the supported range of frequencies and duty resolutions](https://docs.espressif.com/projects/esp-idf/en/v4.4/esp32/api-reference/peripherals/ledc.html?#supported-range-of-frequency-and-duty-resolutions) before proceeding.
```

Comment:
I'd rephrase this to sound less ominous. e.g. Please refer to espressif documentation<link> for valid frequency and duty resolutions.

Reply:
@gvaradarajan Not a blocker: this addition is great and should be committed asap. We should follow up (e.g. open a ticket) with more user-friendly instructions on valid values -- espressif docs are really in the weeds and refer to apis that we obscure.